src/Categories/Bicategory/Construction/Bimodules/Tensorproduct/Coherence/Pentagon.agda

```diff
@@ -0,0 +1,709 @@
+open ComposeWithLocalCoequalizer 𝒞 localCoeq
+open import Categories.Bicategory.Construction.Bimodules.Tensorproduct {o} {ℓ} {e} {t} {𝒞} {localCoeq}
+private
+  _⊗₀_ = TensorproductOfBimodules.B₂⊗B₁
+  _⊗₁_ = TensorproductOfHomomorphisms.h₂⊗h₁
+  
+infixr 30 _⊗₀_ _⊗₁_
+
+import Categories.Bicategory.Extras as Bicat
+open Bicat 𝒞
+
+import Categories.Diagram.Coequalizer
+import Categories.Morphism
+
+-- To get constructions of the hom-categories with implicit arguments into scope --
+private
+  module HomCat {X} {Y} where
+    open Categories.Morphism (hom X Y) public using (_≅_)
+    open Categories.Diagram.Coequalizer (hom X Y) public
+
+open HomCat
+
+open import Categories.Bicategory.Construction.Bimodules.Tensorproduct.Associator
+  {𝒞 = 𝒞} {localCoeq}
+  using (Associator⊗From; hexagon)
+
+open TensorproductOfHomomorphisms using (αSq)
+
+open Bimodule B₁ using () renaming (F to F₁)
+open Bimodule B₂ using () renaming (F to F₂)
+open Bimodule B₃ using () renaming (F to F₃)
+open Bimodule B₄ using () renaming (F to F₄)
+open TensorproductOfBimodules B₂ B₁ using (F₂⊗F₁)
+open TensorproductOfBimodules B₃ B₂ using () renaming (F₂⊗F₁ to F₃⊗F₂)
+open TensorproductOfBimodules B₄ B₃ using () renaming (F₂⊗F₁ to F₄⊗F₃)
+open TensorproductOfBimodules B₃ (B₂ ⊗₀ B₁) using () renaming (F₂⊗F₁ to F₃⊗F₂⊗F₁)
+open TensorproductOfBimodules (B₃ ⊗₀ B₂) B₁ using () renaming (F₂⊗F₁ to [F₃⊗F₂]⊗F₁)
+open TensorproductOfBimodules (B₄ ⊗₀ B₃) B₂ using () renaming (F₂⊗F₁ to [F₄⊗F₃]⊗F₂ )
+open TensorproductOfBimodules B₄ (B₃ ⊗₀ B₂) using () renaming (F₂⊗F₁ to F₄⊗F₃⊗F₂ )
+open TensorproductOfBimodules ((B₄ ⊗₀ B₃) ⊗₀ B₂) B₁ using () renaming (F₂⊗F₁ to [[F₄⊗F₃]⊗F₂]⊗F₁)
+open TensorproductOfBimodules (B₄ ⊗₀ B₃) (B₂ ⊗₀ B₁) using () renaming (F₂⊗F₁ to [F₄⊗F₃]⊗F₂⊗F₁)
+open TensorproductOfBimodules (B₄ ⊗₀ B₃ ⊗₀ B₂) B₁ using () renaming (F₂⊗F₁ to [F₄⊗F₃⊗F₂]⊗F₁)
+open TensorproductOfBimodules B₄ ((B₃ ⊗₀ B₂) ⊗₀  B₁) using () renaming (F₂⊗F₁ to F₄⊗[F₃⊗F₂]⊗F₁)
+open TensorproductOfBimodules B₄ (B₃ ⊗₀ B₂ ⊗₀ B₁) using () renaming (F₂⊗F₁ to F₄⊗F₃⊗F₂⊗F₁)
+
+abstract
+  -- We reduce the pentagon law for the tensorproduct to the pentagon law in 𝒞 --
+  -- For this, we consider a prism with the following five faces. --
+
+  face[[43]2]1⇒[43]21 : Bimodhom.α (Associator⊗From {B₃ = B₄ ⊗₀ B₃} {B₂} {B₁})
+                        ∘ᵥ Coequalizer.arr [[F₄⊗F₃]⊗F₂]⊗F₁
+                        ∘ᵥ Coequalizer.arr [F₄⊗F₃]⊗F₂ ◁ F₁
+                        ∘ᵥ Coequalizer.arr F₄⊗F₃ ◁ F₂ ◁ F₁
+                        ≈ (Coequalizer.arr [F₄⊗F₃]⊗F₂⊗F₁
+                          ∘ᵥ Coequalizer.obj F₄⊗F₃ ▷ Coequalizer.arr F₂⊗F₁
+                          ∘ᵥ Coequalizer.arr F₄⊗F₃ ◁ (F₂ ∘₁ F₁))
+                          ∘ᵥ associator.from {f = F₄ ∘₁ F₃} {F₂} {F₁}
+  face[[43]2]1⇒[43]21 = begin
+  
+    Bimodhom.α (Associator⊗From {B₃ = B₄ ⊗₀ B₃} {B₂} {B₁})
+    ∘ᵥ Coequalizer.arr [[F₄⊗F₃]⊗F₂]⊗F₁
+    ∘ᵥ Coequalizer.arr [F₄⊗F₃]⊗F₂ ◁ F₁
+    ∘ᵥ Coequalizer.arr F₄⊗F₃ ◁ F₂ ◁ F₁
+    ≈⟨ sym-assoc₂ ⟩
```

Comment:
have you experimented with indenting the justification lines a lot more?  Since these are all quite short, you could create a 2-column view (but keep the first column as you have it.)